### PR TITLE
Kanister profile

### PR DESCRIPTION
**Pull request by gauthamsunjay**
**What this PR does / why we need it**:

* Provides the option to install a profile CR along with the operator
* the profile CR is installed via a subchart

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

* Fixes the issue raised in kanister - https://github.com/kanisterio/kanister/issues/82

**Special notes for your reviewer**:

Are we planning to move this chart to the kanister repo?

### CODE REVIEW

File: stable/kanister-operator/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: profile
+    version: 0.1.0
```
Comment (tdmanv) [inline stable/kanister-operator/requirements.yaml:3]: This can't be templated, correct? What happens when we update the profile chart?

Comment (depohmel) [reply stable/kanister-operator/requirements.yaml:3]: from https://github.com/kubernetes/helm/blob/master/docs/helm/helm_dependency.md
``` The 'version' field should contain a semantic version or version range. ```

**Comment by depohmel** [comment]
To push it upstream you would have to bump chart version. Also @ntolia wanted to drop v form chart version

File: stable/kanister-operator/requirements.yaml (hunk 1 above)
Comment (depohmel) [reply stable/kanister-operator/requirements.yaml:3]: something like  version: =>0.1.0  should work

Comment (gauthamsunjay) [reply stable/kanister-operator/requirements.yaml:3]: Yeah, we can do that..https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/conventions.md

**Comment by gauthamsunjay** [comment]
Sure, Ill do that. What should I change the chart version to?

**Comment by depohmel** [comment]
closing since we have our chart repo added into https://hub.helm.sh/

**Pull request closed without merging**